README.md

```diff
@@ -1 +1,122 @@
+
+For more references and description, see https://www.openmicroscopy.org/site/support/ome-model/ome-tiff/data.html.
+
+## Benchmark tests
+
+For each of the datasets above, four benchmark tests were executed:
+
+-   metadata.read: the metadata is extracted from the OME-TIFF tag and
+    converted into the OME Data Model
+-   metadata.write: the metadata is written to disk as an OME-XML file
+-   pixeldata.read: the pixeldata is read from the OME-TIFF and stored in
+    memory
+-   pixeldata.write: the pixeldata is written to disk as another OME-TIFF
+
+For each test, the execution time is measured in the benchmark script
+using the standard system utility function and stored in a tabular form.
+
+## Building and executing the benchmark scripts
+
+### Windows
+
+The benchmark scripts have been executed on a Windows 2008 Server. The other
+build requirements are [Cmake](https://cmake.org/), 
+[Maven](http://maven.apache.org/),
+[Visual Studio](https://www.visualstudio.com/) and a local version of the 
+standalone OME Files bundle for the chosen Visual Studio version.
+
+In the context of our benchmark, we used
+[Jenkins](https://jenkins.io/index.html) to trigger the Windows benchmark
+builds. The building and execution script is available under [jenkins_build.bat]([scripts/jenkins_build.bat).
+
+To build the OME Files performance scripts manually, within a `build` directory
+run the following `cmake` command:
+
+    $ cmake -G "Ninja" -DCMAKE_VERBOSE_MAKEFILE:BOOL=%verbose%
+      -DCMAKE_INSTALL_PREFIX:PATH=%installdir% -DCMAKE_BUILD_TYPE=%build_type%
+     "-DCMAKE_PREFIX_PATH=%OME_FILES_BUNDLE%" 
+      -DCMAKE_PROGRAM_PATH=%OME_FILES_BUNDLE%\bin
+      -DCMAKE_LIBRARY_PATH=%OME_FILES_BUNDLE%\lib 
+      -DBOOST_ROOT=%OME_FILES_BUNDLE% %sourcedir% 
+    $ cmake --build .
+    $ cmake --build . --target install
+
+The Bio-Formats performance script can be built within the source directory
+using Maven:
+
+    $ cd source
+    $ call mvn clean install
+
+### Linux
+
+The Linux benchmark was performed on Ubuntu 16.04. To ease the distribution and
+reproducibility of the suite, the benchmark environment is built using
+[Docker](https://www.docker.com/) via a [Dockerfile](Dockerfile). To build the
+benchmark Docker image, run:
+
+    $ docker build -t ome-files-performance .
+
+In order to execute the benchmark scripts, download the benchmark datasets
+under a local folder e.g. `/tmp/benchmark_data` then mount this local folder as
+a  `/data` volume and run the Docker image:
+
+    $ docker run --rm -it -v /data:/data ome-files-performance
+
+This will execute the [run_benchmarking](scripts/run_benchmarking) script and
+store the output of the benchmark under `/data/out` and the tabular results
+under `/data/results`.
+
```

Comment:
Do we want a section on running the benchmarks via maven or directly, or just to reference the scripts?